automatic/opera/README.md

```diff
@@ -4,11 +4,12 @@ The Opera web browser makes the Web fast and fun, giving you a better web browse
 
 ## Parameters
 
+- `/NoAutostart` - Do not add Opera entry to `HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Run` registry
```

Comment:
I don't think we need to talk about the registry entry. Just some text that it won't start at login? Something simple.